src/bin/regex_capture_iter_prod_feedback.rs

```diff
@@ -0,0 +1,47 @@
+// captures_iter: allows you to iterate over multiple captures of a 
+// regular expression pattern in a text.
+// The below example captures the email feedbacks and prints it
+
+use regex::Regex;
+
+fn capture_info_from_feedbacks(feedbacks: Vec<&str>) -> Vec<(String, String, String)> {
+    let mut captured_info = Vec::new(); // Store captured information
+
+    // Define a regex pattern to capture product, sentiment, and issues
+    let pattern = r"Product: (\w+), Sentiment: (\w+), Issues: (.+)";
+    let re = Regex::new(pattern).expect("Invalid regex pattern");
+
+    // Extract information using captures_iter
+    for feedback_email in feedbacks {
+        for captures in re.captures_iter(feedback_email) {
+            if let (Some(product), Some(sentiment), Some(issues)) = (
+                captures.get(1).map(|m| m.as_str().to_string()),
```

Comment:
Can you write some tests to prove this works? I am looking at this code and suspect it works in most cases. It probably works only in some specific, happy path cases.